### DISCUSSION TITLE
Error deploying project gatsby

### DISCUSSION THREAD

**Discussion by Crushoverride007** [vercel/community#4549] [Bug Report]
### Page to Investigate

https://vercel.com/crushoverride007/next-gen-portfolio/HQ4xKyLtBiw2pcfQt3Sh4ecYg3au

### Steps to Reproduce

Hello i'm trying to move my project built with gatsby from netlify to vercel, but i keep getting this error error 

"(node:446) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 end listeners added to [PassThrough]. Use emitter.setMaxListeners() to increase limit"

Can someone help please

**Comment by amyegan** [comment]
You can use [Vercel CLI](https://vercel.com/docs/cli) on your local computer or development environment to get more information. If you run [`vercel build --debug`](https://vercel.com/docs/cli/build#) it will output extra debugging details.

If you need more help, I'm happy to take a look as well. Just share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use for debugging.

**Reply by Crushoverride007** [reply]
yes i can't seem to understand where the problem is coming from, everything seems to be good when trying to build and then the it generates me this problem
![CleanShot 2023-10-23 at 17 05 16](https://github.com/vercel/community/assets/45631812/5fd37509-0fca-4538-ad61-5a6a0c593f07)
![CleanShot 2023-10-23 at 17 05 22](https://github.com/vercel/community/assets/45631812/38e175df-0442-4e61-bff8-7d2ae3e8d873)

here's a link to the deployment 

`https://vercel.com/crushoverride007/next-gen-portfolio/717bRNCtGKa2UDY7UuBHdo6Hd7py
`

I would be tremendously grateful if you could help
